### PR TITLE
Feature/microdata

### PR DESCRIPTION
**Pull request by marcelotokarnia**
This PR adds the "microdata" google indexing feature:

Read more in: https://developers.google.com/search/docs/data-types/product

This feature can be tested in public stores, for example https://apps.vtex.com

This branch is linked in my workspace 'tok'

You can test it on the following URL:

https://search.google.com/structured-data/testing-tool/u/0/?hl=pt-BR

When prompted for a url, just insert a product page on the workspace, for example:

https://apps.vtex.com/correioslog/p?workspace=tok

or just access the "full link":

https://search.google.com/structured-data/testing-tool/u/0/?hl=pt-BR#url=https%3A%2F%2Fapps.vtex.com%2Fcorreioslog%2Fp%3Fworkspace%3Dtok

Note it may fail the first time(s), if the pods are sleeping... 
I think it might not be a problem for indexing when the stores are all up and running...
There will always be pods up. What do you think ?

### CODE REVIEW

File: react/components/MicroData.js [hunk 1]
```diff
@@ -0,0 +1,74 @@
+import React, { Fragment } from 'react'
+import PropTypes from 'prop-types'
+import { pathOr, path, map, sort, compose, head } from 'ramda'
+
+const lowestPriceInStockSeller = item => {
+  if (item.sellers.length) {
+    return sort((a, b) => a.commertialOffer && a.commertialOffer.availableQuantity > 0
```
Comment (andreldsa) [inline react/components/MicroData.js]: You could use better variable names, like `item` and `otherItem`.

File: react/components/MicroData.js [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+        : -1
+      : -1,
+    item.sellers)[0]
+  }
+  return null
+}
+
+const lowestPriceItem = compose(
+  head,
+  sort((a, b) => (path(['seller', 'commertialOffer', 'price'], a) - path(['seller', 'commertialOffer', 'price'], b)))
```
Comment (andreldsa) [inline react/components/MicroData.js]: Same

File: react/components/MicroData.js [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+        $<span property="price">{path(['commertialOffer', 'price'], seller)}</span>
+        (Sale ends <time property="priceValidUntil" datetime={path(['commertialOffer', 'priceValidUntil'], seller)}>
+          {path(['commertialOffer', 'priceValidUntil'], seller)}
+        </time>)
+        Available from: <span property="seller" typeof="Organization"
+        >
+          <span property="name">{seller.sellerName}</span>
+        </span>
+        Condition: <link property="itemCondition" href="http://schema.org/NewCondition" />New
+        { pathOr(100, ['commertialOffer', 'availableQuantity'], seller)
```
Comment (andreldsa) [inline react/components/MicroData.js]: Magic number? Why 100?

Comment (marcelotokarnia) [reply react/components/MicroData.js]: Any number bigger than 0. 
Just so the client won't miss the opportunity to see the product if we can't determine its availability \o/

Kinda of random choice though, I must admit.
What do you think ?

File: react/components/MicroData.js (hunk 1 above)
Comment (marcelotokarnia) [reply react/components/MicroData.js]: Guess I can rename those.
Well observed, thanks for the advice :)

File: react/components/MicroData.js (hunk 3 above)
Comment (marcelotokarnia) [reply react/components/MicroData.js]: I will place it in a const named ITEM_AVAILABLE or something 🤷‍♂️

**Comment by marcelotokarnia** [comment]
That's a good practice I always forget about. 
Thanks @brunojdo.
Will do!

**Pull request merged**